client/js/render.js

```diff
@@ -72,6 +75,12 @@ function buildChatMessage(data) {
 	const msg = $(templates[template](data.msg));
 	const text = msg.find(".text");
 
+	if (data.msg.previews.length) {
+		data.msg.previews.forEach((preview) => {
```

Comment:
> That extra return is unwanted.

Yeah, but it doesn't cause any problems. I wouldn't have an issue with this, tbh.